remote/analyze_handler.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python2
+
+import os
+import logging
+import subprocess
+import zipfile
+import redis
+import sched
+import time
+
+from enum import Enum
+
+LOGGER = logging.getLogger('ANALYZE HANDLER')
+LOGGER.setLevel(logging.INFO)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter(
+    '%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+LOGGER.addHandler(ch)
+
+
+class AnalyzeStatus(Enum):
+    ID_PROVIDED = 'ID_PROVIDED'
+    QUEUED = 'QUEUED'
+    ANALYZE_IN_PROGRESS = 'ANALYZE_IN_PROGRESS'
+    ANALYZE_COMPLETED = 'ANALYZE_COMPLETED'
+
+
+def preAnalyze(analyze_id, part_number):
+    LOGGER.info('Pre Analyze Step')
+
+    file_name = 'source'
+    file_extension = '.zip'
+
+    file_path = os.path.join('/workspace', analyze_id)
```

Comment:
I think the `workspace` path could be an argument for the whole script.

Reply:
Done